linode/manager
# M3-5543: Add "Upgrade Pending" chip to Volumes

## Description

Add a "Upgrade Pending" chip to Volumes that have been queued for upgrade to NVMe by the user.

## How to test

1. Switch to a dev environment and go to `/volumes`
2. If you have Volumes that were created before NVMe was available, skip the next step
3. Otherwise, use `curld` to create an attached Volume that uses HDD
4. Schedule a migration in alpha
5. The Volume should now have the "Upgrade to NVMe" chip next to it
6. Click the chip and go through the flow on the attached Linodes Detail page
7. If you go back to `/volumes`, the chip should now say "Upgrade Pending"
8. The page should refresh once it finishes migrating which might take several minutes

Feel free to message me for help in particular for creating an eligible Volume and scheduling a migration
